MonoGame.Framework/Vector3.cs

```diff
@@ -522,7 +530,7 @@ public bool Equals(Vector3 other)
         /// <returns>Hash code of this <see cref="Vector3"/>.</returns>
         public override int GetHashCode()
         {
-            return (int)(this.X + this.Y + this.Z);
+            return X.GetHashCode() + Y.GetHashCode() + Z.GetHashCode();
```

Comment:
> I could - thats my distinction from you.

Again... you are not understanding me.  You cannot contribute code... however you may have changed it around... that you do not own.  This is a rule that *all* open source projects have.

By telling us you are copying code from XNA we have no choice but to do this.  I don't want to ...  but I have to because it has to potential to hurt the project.